test/extended/builds/forcepull.go

```diff
@@ -1,183 +1,179 @@
 package builds
 
-/*
-This particular builds test suite is not part of the "default" group,  because its testing
-centers around manipulation of images tags to confirm whether the `docker pull` invocation occurs
-correctly based on `forcePull` setting in the BuildConfig, and rather than spend time creating / pulling down separate, test only,
-images for each test scenario, we reuse existing images and ensure that the tests do not run in parallel, and step on each
-others toes by tagging the same, existing images in ways which conflict.
-*/
-
 import (
 	"fmt"
 
 	g "github.com/onsi/ginkgo"
 	o "github.com/onsi/gomega"
 
-	"time"
-
+	dockerClient "github.com/fsouza/go-dockerclient"
 	exutil "github.com/openshift/origin/test/extended/util"
 )
 
-var (
-	resetData map[string]string
+const (
+	buildPrefixFS = "ruby-sample-build-fs"
+	buildNameFS   = buildPrefixFS + "-1"
+	buildPrefixTS = "ruby-sample-build-ts"
+	buildNameTS   = buildPrefixTS + "-1"
+	buildPrefixFD = "ruby-sample-build-fd"
+	buildNameFD   = buildPrefixFD + "-1"
+	buildPrefixTD = "ruby-sample-build-td"
+	buildNameTD   = buildPrefixTD + "-1"
+	buildPrefixFC = "ruby-sample-build-fc"
+	buildNameFC   = buildPrefixFC + "-1"
+	buildPrefixTC = "ruby-sample-build-tc"
+	buildNameTC   = buildPrefixTC + "-1"
+
+	corruptor = "docker.io/openshift/origin-custom-docker-builder"
```

Comment:
how is this a corruptor for a custom buildconfig?  isn't this a valid custom builder image?

i guess in the custom case (and arguably all cases) you don't care what the corruptor is as long as it's different from the image the buildconfig should be pulling, but it might be less confusing to pick something that's not a builder here.

docker.io/openshift/origin-base should be quick to pull since it should already exist.